wiki/Tournaments/CHT/1/en.md

```diff
@@ -0,0 +1,476 @@
+---
+tags:
+  - CHT
+---
+
+# Coffee Hispanic Tournament
+
+![CHT logo](img/logo.jpg)
```

Comment:
```suggestion
![CHT Logo](img/logo.jpg)
```

Reply:
Isn't this suppose to be sentence cased too?

Reply:
nevermind then, can be left as it is now

Reply:
nvm, after some consulting we decide to translate it ✌